abitutorials/structure.ipynb

```diff
@@ -3157,8 +3157,8 @@
       "    irefine             Refine structure with abi_sanitize iteratively, stop\n",
       "                        if target space group is obtained.\n",
       "    conventional        Gives a structure with a conventional cell according\n",
-      "                        to certain standards. The standards are defined in\n",
-      "                        doi:10.1016/j.commatsci.2010.05.010\n",
+      "                        to certain standards. The standards are defined by\n",
+      "                        [Setyawan2010](https://doi.org/10.1016/j.commatsci.2010.05.010)\n",
```

Comment:
This section of the jupyter notebook is automatically produced by executing:

      !abistruct.py --help

so one should change the source code of abistruct.py in the AbiPy repo
else these changes will be overwritten the next time I regenerate the notebook.

Reply:
OK, I overlooked this (obvious) point.
I have now made a slightly different modification to abistruct.py !
Best,